src/components/Review/ReviewWrite.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useRef } from "react";
+import styled from "styled-components";
+
+interface ReviewWriteProps {
+  reviewText: string;
+  handleReviewText: (newReviewText: string) => void;
+}
+export default function ReviewWrite(props: ReviewWriteProps) {
+  const { reviewText, handleReviewText } = props;
+  const reviewTextRef = useRef<HTMLTextAreaElement>(null);
+  // 리뷰 텍스트 핸들링
+  const handleTextChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    if (reviewTextRef.current) {
+      reviewTextRef.current.value = e.target.value;
+      console.log(reviewTextRef.current.value.length);
+      handleReviewText(reviewTextRef.current.value);
+    }
+  };
```

Comment:
useRef 사용한 이유가 따로 있을까 ??

```suggestion
  // 리뷰 텍스트 핸들링
  const handleTextChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
    if (reviewText.length >= 1500) return;
    handleReviewText(e.target.value);
  };
```

Reply:
헙,,! 그게 맞네영,,,, ㅠㅡㅠ 값만담을 상자가 필요하다고생각햇는데 ㅎㅅㅎ 실슈 ㅎㅎ